libraries/apollo-mockserver/src/commonTest/kotlin/com/apollographql/apollo3/mockserver/test/EnqueueTest.kt

```diff
@@ -11,6 +12,7 @@ import kotlinx.coroutines.flow.flowOf
 import okio.ByteString.Companion.encodeUtf8
 import kotlin.test.Test
 
+@ApolloInternal
```

Comment:
Not 100% sure why we need those `@ApolloInternal` all around the place given that we Opt-In globally in `CompilerOptions.kt`. 

Without them, the code compiles without warning but the IDE is not happy

Reply:
Found it! https://github.com/apollographql/apollo-kotlin/pull/5308#discussion_r1361983482